domain/learningandworkprogress/src/main/kotlin/uk/gov/justice/digital/hmpps/domain/learningandworkprogress/review/ReviewSchedule.kt

```diff
@@ -35,22 +35,27 @@ enum class ReviewScheduleCalculationRule {
   PRISONER_UN_SENTENCED,
 }
 
-enum class ReviewScheduleStatus(val inScope: Boolean) {
+enum class ReviewScheduleStatus(val inScope: Boolean, val isExclusion: Boolean = false, val isExemption: Boolean = false) {
```

Comment:
Nice addition of the isExclusion property 👍 

We might be able to remove the `inScope` property at some point ... I don't think anything currently uses it. Whilst the concept of a Review Schedule being "in or out of scope" is a thing (hence I added it initially), I _think_ now its a concept that is closer to the MN/Curious implementation than ours. Chances are it's not something we need so can probably remove at some stage 👍